web/static/pages/config/list/config-list.mjs

```diff
@@ -68,6 +68,7 @@ class ConfigList extends StyledLitElement {
   }
 
   render() {
+    console.log('Read cookie:', this.message);
```

Comment:
this wont do anything

Reply:
you probably want getCookieMessage() in a service that this can use & show.

Reply:
Leftover from debugging. Removed it.